common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java

```diff
@@ -18,8 +21,28 @@
 @Slf4j
 public class SolrQueryConfigFileReader {
     private static final String COMMENT_PREFIX = "#";
+    private static final String SPRING_CONFIG_LOCATION = "spring.config.location";
     private final SolrQueryConfig.SolrQueryConfigBuilder builder;
```

Comment:
Where and how are we setting spring.config.location?

Reply:
in the Dockerfile, i.e., the container uses `spring.config.location` to read app properties from config/meta/

Reply:
yes, all good. please merge